Kosher Fails to Run With Error

**Describe the bug**
Fails to run with error.

**To Reproduce**
Steps to reproduce the behavior:
1. Attempt to execute `kosher run` on any project

**Expected behavior**
Runs as usual.

**Screenshots, Configuration Files, Feature Files**
```bash
2019/04/10 15:23:58 Web Driver server [chrome] created. Serving at [http://127.0.0.1:46317].
panic: reflect: call of reflect.Value.Type on zero Value

goroutine 1 [running]:
reflect.Value.Type(0x0, 0x0, 0x0, 0x47, 0x924bc0)
	/usr/local/go/src/reflect/value.go:1743 +0x16c
github.com/cbush06/kosher/vendor/github.com/DATA-DOG/godog.(*Suite).Step(0xc0002d5560, 0x911c60, 0xa72990, 0x0, 0x0)
	/home/cbush/go/src/github.com/cbush06/kosher/vendor/github.com/DATA-DOG/godog/suite.go:100 +0x14a
github.com/cbush06/kosher/steps/websteps.BuildGoDogSuite(0xc0001a68c0, 0xc000129560, 0xc0002d5560)
	/home/cbush/go/src/github.com/cbush06/kosher/steps/websteps/web_steps.go:19 +0x1c8
github.com/cbush06/kosher/cmd.buildFeatureContext(0xc0001a68c0, 0xc000129560, 0xc0002d5560)
	/home/cbush/go/src/github.com/cbush06/kosher/cmd/run.go:130 +0x2e3
github.com/cbush06/kosher/cmd.glob..func3.1(0xc0002d5560)
	/home/cbush/go/src/github.com/cbush06/kosher/cmd/run.go:104 +0x40
github.com/cbush06/kosher/vendor/github.com/DATA-DOG/godog.(*runner).run(0xc0002bf640, 0x14)
	/home/cbush/go/src/github.com/cbush06/kosher/vendor/github.com/DATA-DOG/godog/run.go:75 +0xa0
github.com/cbush06/kosher/vendor/github.com/DATA-DOG/godog.RunWithOptions(0xc000186970, 0x6, 0xc00021c270, 0x0, 0x0, 0x0, 0x0, 0x0, 0x9c7e8e, 0x8, ...)
	/home/cbush/go/src/github.com/cbush06/kosher/vendor/github.com/DATA-DOG/godog/run.go:173 +0x7b7
github.com/cbush06/kosher/cmd.glob..func3(0xe0ff40, 0xe36850, 0x0, 0x0, 0x0, 0x0)
	/home/cbush/go/src/github.com/cbush06/kosher/cmd/run.go:103 +0x7ad
github.com/cbush06/kosher/vendor/github.com/spf13/cobra.(*Command).execute(0xe0ff40, 0xe36850, 0x0, 0x0, 0xe0ff40, 0xe36850)
	/home/cbush/go/src/github.com/cbush06/kosher/vendor/github.com/spf13/cobra/command.go:762 +0x473
github.com/cbush06/kosher/vendor/github.com/spf13/cobra.(*Command).ExecuteC(0xe0fa80, 0x9e6387, 0x44, 0xe0ff40)
	/home/cbush/go/src/github.com/cbush06/kosher/vendor/github.com/spf13/cobra/command.go:852 +0x2fd
github.com/cbush06/kosher/vendor/github.com/spf13/cobra.(*Command).Execute(0xe0fa80, 0xe0fa80, 0x1)
	/home/cbush/go/src/github.com/cbush06/kosher/vendor/github.com/spf13/cobra/command.go:800 +0x2b
github.com/cbush06/kosher/cmd.Init()
	/home/cbush/go/src/github.com/cbush06/kosher/cmd/commands.go:21 +0x172
main.main()
	/home/cbush/go/src/github.com/cbush06/kosher/main.go:8 +0x20
```

**Desktop (please complete the following information):**
 - OS: Windows / Linux
 - Browser: Chrome / Firefox / etc.
 - Version: 1.3.0

--- a/common/current_version.go
+++ b/common/current_version.go
@@ -4,5 +4,5 @@ package common
 var CurrentVersion = &Version{
 	Major: 1,
 	Minor: 3,
-	Patch: 0,
+	Patch: 1,
 }

--- a/izpack.xml
+++ b/izpack.xml
@@ -4,7 +4,7 @@
 
     <info>
         <appname>Kosher</appname>
-        <appversion>1.3.0</appversion>
+        <appversion>1.3.1</appversion>
         <appsubpath>kosher</appsubpath>
         <run-privileged/>
         <uninstaller write="yes" />

--- a/steps/websteps/browser_steps.go
+++ b/steps/websteps/browser_steps.go
@@ -94,6 +94,8 @@ func iSwitchToWindowIndex(s *steputils.StepUtils, nth string, idx int) error {
 	return s.Page.Session().SetWindow(windows[idx])
 }
 
-func iReloadThePage(s *steputils.StepUtils) error {
-	return s.Page.Refresh()
+func iReloadThePage(s *steputils.StepUtils) func() error {
+	return func() error {
+		return s.Page.Refresh()
+	}
 }

--- a/suitecontext/suitecontext.go
+++ b/suitecontext/suitecontext.go
@@ -1,0 +1,41 @@
+package suitecontext
+
+import (
+	"github.com/DATA-DOG/godog"
+	"github.com/DATA-DOG/godog/gherkin"
+)
+
+// SuiteContext stores the status of test suite execution
+// by means of registered callbacks with GoDog.
+type SuiteContext struct {
+	StepsPassed        int
+	StepsFailed        int
+	StepsSkipped       int
+	StepsUndefined     int
+	FeaturesPassed     int
+	FeaturesFailed     int
+	FeaturesIncomplete int
+}
+
+// CreateSuiteContext creates a new SuiteContext struct,
+// registeres associated callbacks with GoDog, and returns
+// a reference to the new struct.
+func CreateSuiteContext(s *godog.Suite) *SuiteContext {
+	s.BeforeFeature(beforeFeature)
+	s.BeforeScenario(beforeScenario)
+	s.BeforeStep(beforeStep)
+
+	return &SuiteContext{}
+}
+
+func beforeFeature(f *gherkin.Feature) {}
+
+func beforeScenario(s interface{}) {}
+
+func beforeStep(s *gherkin.Step) {}
+
+func afterFeature(f *gherkin.Feature) {}
+
+func afterScenario(s interface{}, e error) {}
+
+func afterStep(s *gherkin.Step, e error) {}

